condor_git_config.py

```diff
@@ -196,7 +196,7 @@ def _refresh(self):
         else:
             try:
                 subprocess.check_output(
-                    ["git", "pull"], timeout=30, cwd=repo_path, universal_newlines=True
+                    ["git", "pull", "--ff-only"], timeout=30, cwd=repo_path, universal_newlines=True
```

Comment:
This was you should make the linter happy as well. ;)
```suggestion
                    ["git", "pull", "--ff-only"],
                    timeout=30,
                    cwd=repo_path,
                    universal_newlines=True,
                )
```